docs/release-notes/8.5.asciidoc

```diff
@@ -0,0 +1,82 @@
+* Updates the *Anomalies* tab to display the same quantity of anomalies when navigating from the Entity Analytics dashboard ({pull}139910[#139910]).
+* Enriches alerts with host and user risk scores ({pull}139478[#139478]).
+* Enables the Indicators page by default and makes the functionality generally available ({pull}141117[#141117]).
+* Allows indicator data to be investigated in Timeline by including the *Add to Timeline* button throughout the Indicators table ({pull}138836[#138836], {pull}140496[#140496]).
+* Removes the Host risk score card from the Overview dashboard ({pull}140177[#140177]).
+* Adds the option to bulk edit rule schedules to the bulk actions menu in the Rules table ({pull}140166[#140166]).
+* Adds the option to bulk edit rule actions to the bulk actions menu in the Rules table ({pull}138900[#138900]).
+* Adds an alert count card to the User, Host, and Network detail pages. The card shows alerts per rule and can be filtered by alert status ({pull}140150[#140150]).
+* Allows users to examine alerts associated with events and enables the Alerts related by process ancestry section by default if they have a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
+* Enables the Alerts related by session ID section by default. It appears in the Alert details flyout if users have a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
+* Renames the Elastic Endpoint and Cloud Security integration to the Elastic Defend integration ({pull}139517[#139517]).
+* Adds preconfigured use-cases to the setup wizard for the Elastic Defend integration (formerly known as Endpoint and Cloud Security), each with different default settings ({pull}139230[#139230]).
+* Updates the UI for the rule details page's *Exceptions* tab ({pull}138770[138770]).
+* Enables the Osquery Response Action on custom query detection rules and adds an *Osquery Results* tab to the Alert details flyout. Users can use the Osquery Response Action to immediately query hosts that generate alerts ({pull}133279[#133279]).
+* Enables rule exceptions to reference value lists, regardless of rule type. One caveat is that text type value lists still do not work for EQL and threshold rules ({pull}133254[#133254]).
+* Introduces the new alert renderer, which concisely displays a detailed summary of the `kibana.alert.reason` field. It appears in Timeline, throughout the Alerts page, and on the Alert details flyout ({pull}140825[#140825]).
+* Kubernetes Security Posture Management (KSPM) is now GA. You can use the integration to monitor the security posture of your self-managed and Amazon EKS clusters at scale in production. 
+
+[discrete]
+[[bug-fixes-8.5.0]]
+==== Bug fixes and enhancements
+** Endpoint response actions console UI indicates if response action commands aren't supported by installed version of {agent} ({pull}138662[#138662]).
```

Comment:
This alignment is off in the preview - prob need to remove the extra `*`. 
![2022-10-18_12-31-42](https://user-images.githubusercontent.com/57149392/196489547-7e39a875-68bb-43c2-9363-9d9552c23d21.png)

Reply:
Should be fixed now!